addons/binding/org.openhab.binding.rotelra1x/src/main/java/org/openhab/binding/rotelra1x/internal/ConfigurationError.java

```diff
@@ -0,0 +1,14 @@
+package org.openhab.binding.rotelra1x.internal;
+
+public class ConfigurationError extends Exception {
```

Comment:
Missing class level docblock, it should describe its use and who wrote it, an author tag